docs/core/porting/github-copilot-app-modernization/how-to-upgrade-with-github-copilot.md

```diff
@@ -1,154 +1,213 @@
 
-- Review the test results and address the single failing test in `WpfMovie.Tests`.
-- Ensure all updated NuGet packages are compatible with your application.
-- Leverage new features and improvements in .NET 9.0 for your projects.
+- [ ] (1) Update TargetFramework to net10.0 in MvcMovie.csproj, MvcMovie.Tests.csproj, RazorMovie.csproj, RazorMovie.Tests.csproj per Plan §Phase 1
+- [ ] (2) Update TargetFramework to net10.0-windows in WpfMovie.csproj, WpfMovie.Tests.csproj per Plan §Phase 1
+- [ ] (3) All project files updated to target framework (**Verify**)
+- [ ] (4) Update package references per Plan §Package Update Reference (MvcMovie: EF Core 10.0.1, Code Generation 10.0.0; RazorMovie: HtmlSanitizer 9.0.889 security fix)
+- [ ] (5) All package references updated (**Verify**)
+- [ ] (6) Restore all dependencies across solution
+- [ ] (7) All dependencies restored successfully (**Verify**)
+- [ ] (8) Build solution and fix all compilation errors per Plan §Breaking Changes Catalog (focus: BinaryFormatter removal in WpfMovie, WPF control API binary incompatibilities, UseExceptionHandler behavioral changes, HtmlSanitizer API changes)
+- [ ] (9) Solution builds with 0 errors (**Verify**)
+- [ ] (10) Commit changes with message: "TASK-002: Complete atomic upgrade to .NET 10 (all projects, packages, and compilation fixes)"
+
+---
+
+### [ ] TASK-003: Run full test suite and validate upgrade
+**References**: Plan §Phase 2, Plan §Testing & Validation Strategy
+
+- [ ] (1) Run tests in MvcMovie.Tests, RazorMovie.Tests, and WpfMovie.Tests projects
+- [ ] (2) Fix any test failures (reference Plan §Breaking Changes Catalog for common issues: HtmlSanitizer behavior, BinaryFormatter replacement, framework behavioral changes)
+- [ ] (3) Re-run all tests after fixes
+- [ ] (4) All tests pass with 0 failures (**Verify**)
+- [ ] (5) Commit test fixes with message: "TASK-003: Complete testing and validation for .NET 10 upgrade"
 ```
 
-## Next steps
+To run the upgrade:
+
+1. Tell Copilot to start the upgrade.
+1. Monitor progress by reviewing the `tasks.md` file as Copilot updates task statuses.
+1. If Copilot encounters a problem it can't resolve, provide the requested help.
+1. Let Copilot learn from your interventions and continue the upgrade.
+
+The tool creates a Git commit for every portion of the process, so you can easily roll back changes or get detailed information about what changed.
+
+## Verify the upgrade
+
+When the upgrade finishes, Copilot shows next steps in the chat response to guide you on what to do after the process. Review the `tasks.md` file for the status of every step. The tool creates a Git commit for every portion of the upgrade process, so you can easily roll back changes or get detailed information about what changed.
+
+The following example shows completed tasks for an ASP.NET Core project upgrade:
+
+```markdown
```

Comment:
I noticed you mixed `md` and `markdown` as the marker for this raw markdown content. I'd pick one for consistency.